lib/fluent/logger/unix_fluent_logger.rb

```diff
@@ -0,0 +1,18 @@
+module Fluent
+  module Logger
+    class UNIXFluentLogger < FluentLogger
```

Comment:
By design, inheriting is bad because unix domain socket is not a subclass of FluentLogger.
So adding option to FluentLogger is better.